Automattic/wp-calypso
# Login: Add i18n support for client side rendering of Login page

Implements #14506.

We completely missed i18n support for all login pages. This PR tries to fix it. Implementation is going to be similar to what we have at the moment in the Signup flow.

### TODO
* [x] Disable server side rendering when non-english locale selected 
* [x] Show a locale suggestions box whenever applicable.
* [x] Update 2FA login flow to respect the selected locale.
* [x] Fix wrong urls in Locale Suggestion box when no locale provided in url.

### Testing

1. Open http://calypso.localhost:3000/log-in.
2. It should load Login page in English as it is now.
3. Open http://calypso.localhost:3000/log-in/de.
4. It should load Login page in German.
5. Open http://calypso.localhost:3000/log-in/he.
6. It should load Login page in Hebrew and RTL direction.
7. Play with locale suggestions. Make sure it is displayed only on the starting Login page.

At the moment only English version is rendered on the server.

### Review


### Known issues
* Missing server side rendering for non-en locales.
* Locale suggestions box doesn't render sever-side. It uses Flux store and probably it can't be triggered when components is rendered on the server.
